int8 ar_id
float32 goal_dist
---
string message
bool success